### PR TITLE
Release Flink ML 2.1.0

### PR DESCRIPTION
**Pull request by zhipeng93**

### CODE REVIEW

**Review by dianfu — APPROVED** [review]

File: _posts/2022-07-07-release-ml-2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+  This change introduces all the breaking changes listed in the Flink 1.15
+  [release notes](https://nightlies.apache.org/flink/flink-docs-release-1.15/release-notes/flink-1.15/).
+
+# Release Notes and Resources
+
+Please take a look at the [release notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12351141)
+for a detailed list of changes and new features.
+
+The binary distribution and source artifacts are now available on the updated
```
Comment (lindong28) [inline _posts/2022-07-07-release-ml-2.1.0.md]: Can you remove the `binary distribution` here?

File: _posts/2022-07-07-release-ml-2.1.0.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+## Algorithm Library
+This release continues to extend the algorithm library in Flink ML, with the 
+focus on validating the functionalities and the performance of Flink ML 
+infrastructure using representative algorithms in different categories.
+
+Below are the lists of algorithms newly supported in this release, grouped by 
+their categories:
+
+- feature engineering (MinMaxScaler, StringIndexer, VectorAssembler,
```
Comment (lindong28) [inline _posts/2022-07-07-release-ml-2.1.0.md]: Would it be better to start the sentence with an upper-case letter?

File: _posts/2022-07-07-release-ml-2.1.0.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+### Tools for benchmarking algorithm performance
+
+An easy-to-use benchmark framework is critical to developing and maintaining 
+performant algorithm libraries in Flink ML. This release added a benchmark 
+framework that provides APIs to write pluggable and reusable data generators, 
+takes benchmark configuration in JSON format, and outputs benchmark results in 
+JSON format to enable custom analysis. An off-the-shelf script is provided to 
+visualize benchmark results using Matplotlib. Feel free to check out this 
+[README](https://github.com/apache/flink-ml/blob/master/flink-ml-benchmark/README.md)
```
Comment (lindong28) [inline _posts/2022-07-07-release-ml-2.1.0.md]: Change the link to https://github.com/apache/flink-ml/blob/release-2.1/flink-ml-benchmark/README.md?

File: _posts/2022-07-07-release-ml-2.1.0.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+- feature engineering (MinMaxScaler, StringIndexer, VectorAssembler, 
+StandardScaler, Bucketizer)
+- online learning (OnlineKmeans, OnlineLogisiticRegression)
+- regression (LinearRegression)
+- classification (LinearSVC)
+- evaluation (BinaryClassificationEvaluator)
+
+Example Python and Java programs for these algorithms are provided on the Apache
+Flink ML [website](https://nightlies.apache.org/flink/flink-ml-docs-stable/) to
```
Comment (lindong28) [inline _posts/2022-07-07-release-ml-2.1.0.md]: Change the link to https://nightlies.apache.org/flink/flink-ml-docs-release-2.1/?

File: _posts/2022-07-07-release-ml-2.1.0.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+- regression (LinearRegression)
+- classification (LinearSVC)
+- evaluation (BinaryClassificationEvaluator)
+
+Example Python and Java programs for these algorithms are provided on the Apache
+Flink ML [website](https://nightlies.apache.org/flink/flink-ml-docs-stable/) to 
+help users learn and try out Flink ML. And we also provided example benchmark 
+[configuration files](https://github.com/apache/flink-ml/tree/master/flink-ml-benchmark/src/main/resources)
+in the repo for users to validate Flink ML performance. Feel free to check out 
+this [README](https://github.com/apache/flink-ml/blob/master/flink-ml-benchmark/README.md)
```
Comment (lindong28) [inline _posts/2022-07-07-release-ml-2.1.0.md]: Change the link to https://github.com/apache/flink-ml/blob/release-2.1/flink-ml-benchmark/README.md?

**Review by lindong28 — COMMENTED** [review]
Thanks for the PR! Left some comments below.

File: _posts/2022-07-07-release-ml-2.1.0.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+Specifically, this release added two online learning algorithm prototypes (i.e. 
+OnlineKMeans and OnlineLogisticRegression) with tests covering the entire 
+lifecycle of using these algorithms. These two algorithms introduce concepts 
+such as global batch size and model version, together with metrics and APIs to 
+set and get those values. While the online algorithm prototypes have not been 
+optimized for prediction accuracy yet, this line of work is an important step 
+toward setting up best practices for building online learning algorithms in 
+Flink ML. We hope more contributors from the community can join this effort.
+
+### Tools for benchmarking algorithm performance
```
Comment (lindong28) [inline _posts/2022-07-07-release-ml-2.1.0.md]: How about changing this to `Algorithm benchmark framework`?

**Pull request merged**